src/plugins/data_explorer/public/components/sidebar/index.tsx

```diff
@@ -59,23 +76,86 @@ export const Sidebar: FC = ({ children }) => {
     }
   }, [indexPatternId, activeDataSources, dataSourceOptionList]);
 
+  const redirectToLogExplorer = useCallback(
+    (dsName: string, dsType: string) => {
+      return application.navigateToUrl(
+        `../observability-logs#/explorer?datasourceName=${dsName}&datasourceType=${dsType}`
+      );
+    },
+    [application]
+  );
+
+  const getRedirectModal = useCallback(
+    (dsName: string, dsType: string) => {
+      return (
+        <EuiModal onClose={() => setModalContent(null)}>
+          <EuiModalHeader>
+            <EuiModalHeaderTitle>
+              <h1>
+                {i18n.translate('dataExplorer.sidebar.LogExplorerRedirectionModalTitle', {
+                  defaultMessage: 'Open in Log Explorer',
```

Comment:
is it logs explorer or log explorer.

Reply:
it's log explorer, I copied the text directly from UX.